tools/run_tests/python_utils/jobset.py

```diff
@@ -357,7 +357,7 @@ def stdout(self=self):
                 if measure_cpu_costs:
                     m = re.search(
                         r'real\s+([0-9.]+)\nuser\s+([0-9.]+)\nsys\s+([0-9.]+)',
-                        stdout())
```

Comment:
Fair enough. Btw, I don't think we're using the "measure_cpu_costs" option anywhere anymore.

Reply:
We're definitely using the flag. That's why I ended up making this change. If no one is using this data anymore though, we should consider just ripping it out entirely.

Reply:
Interesting, do you know where we're using it?

Reply:
[This log](https://source.cloud.google.com/results/invocations/8498d457-2d58-4f4d-8587-fa0b7b0aa2f4/log) includes it:

```
run_tests_python_linux_dbg_native_default_python_alpine: "python tools/run_tests/run_tests.py --use_docker -t -j 16 -x run_tests/python_linux_dbg_native_default_python_alpine/sponge_log.xml --report_suite_name python_linux_dbg_native_default_python_alpine -l python -c dbg --iomgr_platform native --arch default --compiler python_alpine --bq_result_table aggregate_results --measure_cpu_costs --report_multi_target"
```

A quick grep indicates that it's coming from [here](https://github.com/grpc/grpc/blob/91a2db9bc2f0195e649380f5e0f46d5ced8c73a6/tools/run_tests/run_tests_matrix.py#L463).